docs/configuration.md

```diff
@@ -0,0 +1,44 @@
+## Liberty Server Configuration
+
+Liberty server configuration can be specified within the build.gradle file by specifying a configuration directory, specific configuration files, or inline properties. This configuration is copied to the appropriate server directory during the `libertyCreate` task.
+
+| Attribute | Description | Required |
+| --------- | ------------ | ----------|
```

Comment:
I think we need to add a `Type` column to the documentation to specify String or File?  Similar to https://docs.gradle.org/current/userguide/war_plugin.html#sec:war_convention_properties